DigitalLearningSolutions.Data.Tests/DataServices/PasswordResetDataServiceTests.cs

```diff
@@ -15,107 +15,99 @@ namespace DigitalLearningSolutions.Data.Tests.DataServices
 
     public class PasswordResetDataServiceTests

```

Comment:
Sorry, I moved `Can_Create_And_Find_A_Password_Reset_For_User` up to the top - the only actual code change on this file is converting that from two separate tests to a theory.